Sources/BraintreePayPal/BTPayPalError.swift

```diff
@@ -73,7 +78,7 @@ public enum BTPayPalError: Error, CustomNSError, LocalizedError, Equatable {
         case .httpPostRequestError(let error):
             return "HTTP POST request failed with \(error)."
         case .invalidURL(let error):
-            return "An error occured with retrieving a PayPal authentication URL: \(error)"
+            return "An error occurred with retrieving a PayPal URL: \(error)"
```

Comment:
We had a typo here and also are using this error for URLs outside of authentication so updated this string.